gst/nnstreamer/tensor_sparse/tensor_sparse_util.c

```diff
@@ -0,0 +1,231 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * GStreamer / NNStreamer Sparse Tensor support
+ * Copyright (C) 2021 Yongjoo Ahn <yongjoo1.ahn@samsung.com>
+ */
+/**
+ * @file	tensor_sparse_util.c
+ * @date	27 Jul 2021
+ * @brief	Util functions for tensor_sparse encoder and decoder.
+ * @see		https://github.com/nnstreamer/nnstreamer
+ * @author	Yongjoo Ahn <yongjoo1.ahn@samsung.com>
+ * @bug		No known bugs except for NYI items
+ */
+
+#ifndef __GST_TENSOR_SPARSE_ENC_H__
+#define __GST_TENSOR_SPARSE_ENC_H__
```

Comment:
let's change this __GST_TENSOR_SPARSE_ENC_H__ > __GST_TENSOR_SPARSE_UTIL_H__

Reply:
Oops I forgot to remove them. Thanks for pointing out.